packages/providers/anthropic/package.json

```diff
@@ -31,16 +31,17 @@
     "test": "vitest run"
   },
   "devDependencies": {
-    "vitest": "^2.1.5",
     "@llamaindex/core": "workspace:*",
-    "@llamaindex/env": "workspace:*"
+    "@llamaindex/env": "workspace:*",
+    "vitest": "^2.1.5"
   },
   "peerDependencies": {
     "@llamaindex/core": "workspace:*",
     "@llamaindex/env": "workspace:*"
   },
   "dependencies": {
     "@anthropic-ai/sdk": "0.37.0",
-    "remeda": "^2.17.3"
+    "remeda": "^2.17.3",
+    "zod": "^3.25.67"
```

Comment:
note that zod must be a `peerDependency` as currently we're having it as dependency in the core package. So if we upgrade to zod/4 we must do it in core first. 
Doing so we probably should make it an optional peerDependency in core too as users usually bring their own